From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 12 May 2024 09:14:46 +0200
Subject: [PATCH] Fix implicit function declaration

Since musl 1.2.5, musl only provides the prototype for basename(3)
in libgen.h (as mandated by POSIX) and not in strings.h.

Ref: https://gitlab.alpinelinux.org/alpine/aports/-/issues/16106
Upstream-Status: Pending
Origin: Alpine Linux
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 rdma/rdma.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/rdma/rdma.c
+++ b/rdma/rdma.c
@@ -8,6 +8,8 @@
 #include "version.h"
 #include "color.h"
 
+#include <libgen.h> /* for basename(3) */
+
 /* Global utils flags */
 int json;
 
